### PR TITLE
Load Vault token from disk if envar not set

### LINKED ISSUE #4: Load Vault token from file
Sometimes the Vault token can't be made available as an envar, but it can be read from the disk. For example, when using `vault agent` in an init container, the agent dumps the token to a file.

### PR DESCRIPTION
**Pull request by larslevie**
Closes #4

### CODE REVIEW

**Comment by larslevie** [comment]
Would be nice if this could take the file location as an arg, but the current Kustomize implementation doesn't allow for that. I considered using one or more of the Kustomize secret generator `args` as a way to pass in arguments, but decided it wasn't that urgent now and it was added complexity.

File: vault.go [hunk 1]
```diff
@@ -12,10 +14,31 @@ type plugin struct{}
 
 var KVSource plugin
 
-var vaultAddr = os.Getenv("VAULT_ADDR")
-var token = os.Getenv("VAULT_TOKEN")
+func getToken() (string, error) {
+	var t, exists = os.LookupEnv("VAULT_TOKEN")
```
Comment (justinsteffy) [inline vault.go:18]: Huh, it really is `LookupEnv` but `Getenv`. Kind of a mixed bag of `*Env` and `*env`.

Comment (larslevie) [reply vault.go:18]: Yup

**Review by justinsteffy — APPROVED** [review]
We talked about changing the path of where the `.vault_token` lives, but otherwise LGTM.

**Pull request merged**